docs/resources/statement.md

```diff
@@ -0,0 +1,106 @@
+# `jupyter` accessibility statement
+
+
+## `jupyter` audits
+
+This section includes audits from different stakeholders on `jupyter` products.
+
+* [JupyterLab v2.2.6 WCAG 2.1](https://github.com/jupyterlab/jupyterlab/issues/9399)
+* [Jupyter Notebook WCAG 2.0](https://github.com/jupyter/accessibility/issues/7)
+
+## Accessibility Statement for JupyterLab
+
+Edited from the [W3C accessibility statement generator](https://www.w3.org/WAI/planning/statements/generator/#create)
+
+This is an accessibility statement from Jupyter accessibility contributors.
+
+### The current state of JupyterLab
+
+#### Conformance status
+
+The [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag) defines requirements for designers and developers to improve accessibility for people with disabilities. It defines three levels of conformance: Level A, Level AA, and Level AAA. JupyterLab is non conformant with WCAG 2.1 level AA. Non conformant means that the content does not meet the accessibility standard.
+
+![Picture accessibility compliance levels as mountains. The first peak to reach is AA. This includes both A and AA criteria. Behind it, in the mist, there is a larger peak, AAA. That one is not always completely reachable--Marie Guillaumet, Access42](https://stephaniewalter.design/wp-content/uploads/2022/05/stephaniewalter-sommets-de-l-accessibilite.jpg)
+
+*By [Stéphanie Walter](https://stephaniewalter.design) ([Source](https://stephaniewalter.design/blog/5-illustrations-to-understand-and-promote-accessibility/))*
+
+JupyterLab's accessibility does not exist in isolation. JupyterLab inherits much from the many projects it is built upon and its accessibility conformance may impact projects built off of JupyterLab or its components. The accessibility of this ecosystem is interlinked, so conformance may need to be resolved at different levels in order to impact JupyterLab positively.
+
+#### Compatibility with browsers and assistive technology
+
+JupyterLab is designed to be compatible with the following technologies:
+
+* Windows, macOS, iOS, Android; Firefox, Chrome, Safari, Chromium browsers (mobile and desktop).
+
+JupyterLab is not compatible with:
+
+* Internet Explorer, Edge; JAWS, NVDA, VoiceOver, Narrator, Orca screen readers; voice control technology.
+
```

Comment:
@krassowski said
- We are looking to support Edge
- Is it worth mentioning the things we want to support